src/gui/qgsfieldmappingmodel.cpp

```diff
@@ -33,39 +33,43 @@ QVariant QgsFieldMappingModel::headerData( int section, Qt::Orientation orientat
 {
   if ( role == Qt::DisplayRole )
   {
-    if ( orientation == Qt::Horizontal )
+    switch ( orientation )
     {
-      switch ( static_cast<ColumnDataIndex>( section ) )
+      case Qt::Horizontal:
       {
-        case ColumnDataIndex::SourceExpression:
-        {
-          return tr( "Source expression" );
-        }
-        case ColumnDataIndex::DestinationName:
-        {
-          return tr( "Name" );
-        }
-        case ColumnDataIndex::DestinationType:
-        {
-          return tr( "Type" );
-        }
-        case ColumnDataIndex::DestinationLength:
+        switch ( static_cast<ColumnDataIndex>( section ) )
         {
-          return tr( "Length" );
-        }
-        case ColumnDataIndex::DestinationPrecision:
-        {
-          return tr( "Precision" );
-        }
-        case ColumnDataIndex::DestinationConstraints:
-        {
-          return tr( "Constraints" );
+          case ColumnDataIndex::SourceExpression:
+          {
+            return tr( "Source expression" );
+          }
+          case ColumnDataIndex::DestinationName:
+          {
+            return tr( "Name" );
+          }
+          case ColumnDataIndex::DestinationType:
+          {
+            return tr( "Type" );
+          }
+          case ColumnDataIndex::DestinationLength:
+          {
+            return tr( "Length" );
+          }
+          case ColumnDataIndex::DestinationPrecision:
+          {
+            return tr( "Precision" );
+          }
+          case ColumnDataIndex::DestinationConstraints:
+          {
+            return tr( "Constraints" );
```

Comment:
Could you please replace "Constraints" by "Template properties" as agreed in #31387?

Reply:
mmm, I didn't do it on purpose. The widget is not mean to be used only in the refactoring alg, and "Template properties" for a column that contains information about constraints seems rather obscure to me.

@nyalldawson what do you think?

Reply:
In this case it would be best if it's possible to override it from the refactoring algorithm? After all the refactoring algorithm is a very prominent place for this widget.

(For the record, I can also live with "Constraints" but it would be a pity to keep this PR in stale for longer because of this label)

Reply:
> (For the record, I can also live with "Constraints" but it would be a pity to keep this PR in stale for longer because of this label)

Same here. Just mentioned it because it was the consensus here: https://github.com/qgis/QGIS/pull/31387#issuecomment-586851400 and because I understood that the "Template properties" column could have more than constraints, e.g., show if the current field has a relation reference or a default value. But it is up to you, of course.

Reply:
Can we merge for now, and I'll address this in upcoming followups?

Reply:
Gentle bump